src/components/judgments/Judgments.js

```diff
@@ -6,6 +6,7 @@ import * as judgmentRiderConnectionActions from '../../actions/judgmentRiderConn
 import SingleJudgmentContainer from "../../containers/SingleJudgmentContainer";
 import {Timeline, TimelineEvent} from 'react-event-timeline';
 import TimeLineEndBlock from "../common/TimeLineEndBlock";
+import * as riderStageConnectionsActions from '../../actions/riderStageConnectionsActions'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'riderStageConnectionsActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)